scripts/convert.js

```diff
@@ -3,19 +3,18 @@
 // Usage: `convert.js < /path/to/cities.csv > cities.json`
 
 const csv = require('csv');
-const fs = require('fs');
 const mapValues = require('lodash/mapValues');
+const kebabCase = require('lodash/kebabCase');
 const getStdin = require('get-stdin');
-const path = require('path');
+const dataShell = require('../src/constants/data-shell.json');
 
-const dataShellPath = path.resolve(__dirname, '../resources/data-shell.json');
-const dataShell = JSON.parse(fs.readFileSync(dataShellPath, 'utf-8'));
 const dict = mapValues(dataShell.data.indices, (atts) => atts.source);
 
 getStdin().then(rawCsv => {
```

Comment:
This is still a lint error. Are you using a linter plugin for VS Code? It should pick up our `.eslintrc.js` file, but if not you might need to tell it to look there.

Reply:
Ahh, I was assuming CodeClimate would pick up eslint errors. I have now added the plugin, I can see the errors.